mlflow/environment_variables.py

```diff
@@ -637,3 +637,8 @@ def get(self):
 #: set this environment variable to true.
 #: (default: ``True``)
 MLFLOW_ENABLE_DB_SDK = _BooleanEnvironmentVariable("MLFLOW_ENABLE_DB_SDK", True)
+
+#: A flag that's set to 'true' in the child process for capturing modules.
+MLFLOW_IN_CAPTURE_MODULE_PROCESS = _BooleanEnvironmentVariable(
```

Comment:
```suggestion
_MLFLOW_IN_CAPTURE_MODULE_PROCESS = _BooleanEnvironmentVariable(
```

let's make this private. Users don't need to touch this.